examples/torque_driven_ocp/phase_transition_uneven_variable_number.py

```diff
@@ -101,21 +105,22 @@ def prepare_ocp(
         x_bounds=x_bounds,
         u_bounds=u_bounds,
         objective_functions=objective_functions,
+        constraints=constraints,
         variable_mappings=tau_mappings,
         phase_transitions=phase_transitions,
     )
 
+
 def main():
 
     # --- Prepare the ocp --- #
     ocp = prepare_ocp()
 
-    sol = ocp.solve(show_online_optim=True)  # , solver_options={'îpopt.max_iter':1}
+    sol = ocp.solve(show_online_optim=False)
```

Comment:
Unless it causes problem, I'd prefer True here

Reply:
It causes an error with the arrangement of the plots, which I did not intend to fix --'